include/modem/modem_info.h

```diff
@@ -257,6 +257,85 @@ int modem_info_json_object_encode(struct modem_param_info *modem,
  */
 int modem_info_params_get(struct modem_param_info *modem_param);
 
+/** @brief Obtain the UUID of the modem firmware build.
+ *
+ * The UUID is represented as a string, for example:
+ * 25c95751-efa4-40d4-8b4a-1dcaab81fac9
+ *
+ * See the documentation for the AT command %XMODEMUUID.
+ *
+ * @param buf Pointer to the target buffer.
+ * @param buf_size Size of target buffer.
+ *
+ * @retval 0 if the operation was successful.
+ *         Otherwise, a (negative) error code is returned.
+ */
+int modem_info_get_fw_uuid(char *buf, size_t buf_size);
+
+/** @brief Obtain the short software identification.
+ *
+ * The short software identification is represented as a string, for example:
+ * nrf9160_1.1.2
+ *
+ * See the documentation for the AT command %SHORTSWVER.
+ *
+ * @param buf Pointer to the target buffer.
+ * @param buf_size Size of target buffer.
+ *
+ * @retval 0 if the operation was successful.
```

Comment:
What happens if the buffer size provided was not enough?

Reply:
the firmware version will then be truncated